tests/data/big-good.md

```diff
@@ -6,8 +6,22 @@ a = 1
 b = 2
```

Comment:
Is there any objection in having this file in the `good` folder?

Reply:
No objection, I just didn't move it.